xwwp-follow-link.el

```diff
@@ -154,10 +154,12 @@ ACTION should be called with the resulting link.
 UPDATE-FN is a function that can be called when the candidates
 list is narrowed.It will highlight the link list in the
 browser."
-  (funcall action (cdr (assoc (completing-read prompt (lambda (str pred _)
+  (funcall action (cdr (assoc (completing-read prompt (lambda (str pred flag)
                                                         (oset backend text str)
                                                         (funcall update-fn)
-                                                        (try-completion str collection pred))
+                                                        (if (null flag)
```

Comment:
Is there a reason here to use `(if (null flag) ...)` instead of `(if flag ...)` with the clauses inverted ?

Reply:
no reason. I find more suitable function `complete-with-action` for this so I use it.